src/app/lib/config/services/magyarul.ts

```diff
@@ -6,311 +6,392 @@ import { DefaultServiceConfig } from '../../../models/types/serviceConfig';
   default: {
-    lang: 'en-GB',
-    articleAuthor: `https://www.facebook.com/bbcnews`,
-    articleTimestampPrefix: 'Updated',
+    lang: 'hu',
+    articleAuthor: 'https://www.facebook.com/bbcnews',
+    articleTimestampPrefix: 'Frissítve',
     articleTimestampSuffix: '',
-    atiAnalyticsAppName: 'news',
+    atiAnalyticsAppName: 'news-magyarul',
     atiAnalyticsProducerId: '134',
     atiAnalyticsProducerName: 'HUNGARIAN',
-    useReverb: true,
-    chartbeatDomain: 'bbc.co.uk',
-    brandName: 'BBC News',
+    chartbeatDomain: 'magyarul.bbc.co.uk',
+    brandName: 'BBC News Magyarul',
     product: 'BBC News',
     defaultImage:
       'https://static.files.bbci.co.uk/ws/simorgh-assets/public/magyarul/images/metadata/poster-1024x576.png',
-    defaultImageAltText: 'BBC News',
+    defaultImageAltText: 'BBC News Magyarul',
     dir: 'ltr',
-    externalLinkText: ', external',
-    imageCaptionOffscreenText: 'Image caption, ',
-    videoCaptionOffscreenText: 'Video caption, ',
-    audioCaptionOffscreenText: 'Audio caption',
-    defaultCaptionOffscreenText: 'Caption, ',
-    imageCopyrightOffscreenText: 'Image source, ',
-    locale: 'en_GB',
-    datetimeLocale: 'en-gb',
-    service: 'news',
-    serviceName: 'News',
-    languageName: 'English',
+    externalLinkText: ', Külső',
+    imageCaptionOffscreenText: 'Képaláírás, ',
+    videoCaptionOffscreenText: 'Videófelirat, ',
+    audioCaptionOffscreenText: 'Hangfelirat',
+    defaultCaptionOffscreenText: 'Felirat, ',
+    imageCopyrightOffscreenText: 'Kép forrás, ',
+    locale: 'hu-HU',
+    isoLang: 'hu',
+    datetimeLocale: 'hu',
+    service: 'magyarul',
+    serviceName: 'Magyarul',
+    serviceLocalizedName: 'Magyarul',
+    languageName: 'Angol',
     twitterCreator: '@BBCNews',
     twitterSite: '@BBCNews',
-    noBylinesPolicy: 'https://www.bbc.com/news/help-41670342#authorexpertise',
-    publishingPrinciples: 'https://www.bbc.com/news/help-41670342',
+    noBylinesPolicy: 'https://www.bbc.com/magyarul/articles/c8xr72zw50xo',
+    publishingPrinciples: 'https://www.bbc.com/magyarul/articles/c8xr72zw50xo',
     isTrustProjectParticipant: true,
     script: latin,
-    manifestPath:
-      'https://static.files.bbci.co.uk/core/manifest.1ccdbcfd4cc3bf889128a50903c2b22c81758637.json',
-    homePageTitle: 'Home',
+    manifestPath: '/magyarul/manifest.json',
+    swPath: '/magyarul/sw.js',
+    homePageTitle: 'Főoldal',
     showAdPlaceholder: false,
     showRelatedTopics: true,
     translations: {
+      pagination: {
+        page: 'Lap',
+        previousPage: 'Előző oldal',
+        nextPage: 'Következő oldal',
+        pageXOfY: '{x}/{y} oldal',
+      },
       ads: {
-        advertisementLabel: 'Advertisement',
+        advertisementLabel: 'Hirdetés',
+      },
+      byline: {
+        articleInformation: 'Cikk',
+        author: 'Szerző',
+        listItemImage: 'Szerzői kép',
+        published: 'Publikálva',
+        reportingFrom: '',
```

Comment:
We can test what this looks like using: http://localhost:7080/magyarul/articles/crj3r38eex2o?renderer_env=test